src/components/atoms/Loading/index.tsx

```diff
@@ -1,27 +1,62 @@
 "use client";
 
 import { motion } from "framer-motion";
+import { StaticImport } from "next/dist/shared/lib/get-img-props";
 import Image from "next/image";
 import React from "react";
-import LogoWhite from "@/images/easytolive/logo/logobranca-fundoprimary.svg";
+import cx from "classnames";
+
+interface ILoadingComponent {
+  fullSize?: boolean;
+  bgColor?: "main" | "secondary" | "none";
+  Icon?: StaticImport;
+}
+
+const LoadingComponent: React.FC<ILoadingComponent> = ({
+  fullSize = true,
+  bgColor = "main",
+  Icon,
+}) => {
+  const backGroundColor = () => {
```

Comment:
Why is this a function? If it's a function, where's the verb in the name? You can find more here: https://google.github.io/styleguide/jsguide.html#naming-method-names

Also +1 protein bar because Ground shouldn't have G

```suggestion
  const getBackgroundColor = COLORS[bgStyle];
```